### ISSUE TITLE
[Docker Image] Add `jq` by default

### ISSUE THREAD

**Issue by chmuche** [CleverCloud/clever-tools#736]
Now `clever curl` is official, adding `jq` could be great for scripting stuff.

I maintain a dummy image `clevertools-jq` on my private GitLab with only this inside.

```dockerfile
FROM clevercloud/clever-tools:latest
COPY --from=ghcr.io/jqlang/jq /jq /usr/bin/jq
```

**Comment by miton18** [comment]
Are you interested in contributing ?

**Comment by chmuche** [comment]
Yes, I can do a PR on https://github.com/CleverCloud/clever-tools-dockerhub if you want.

I don't if there is some contributing guideline

**Comment by chmuche** [comment]
@miton18 My PR is pushed, and I have fixed a bug with `clever curl` too

https://github.com/CleverCloud/clever-tools-dockerhub/pull/6

**Comment by miton18** [comment]
Hi @chmuche sorry for the late response, I test your PR, it's OK, 
Because this project is overwrite by the clever-tools one, I will make the PR on this one 
https://github.com/CleverCloud/clever-tools/tree/776e024a065e74aa49e63b9e269d809d2ad1676c/templates/dockerhub
